README.md

```diff
@@ -38,29 +38,23 @@ height="80" />](https://f-droid.org/packages/io.github.muntashirakon.AppManager)
 alt="Get it on IzzyOnDroid"
 height="80" />](https://apt.izzysoft.de/fdroid/index/apk/io.github.muntashirakon.AppManager)
 
-App Manager Docs: https://muntashirakon.github.io/AppManager
-
-Telegram Support Group: https://t.me/AppManagerAndroid
-
-Telegram Update Channel: https://t.me/AppManagerChannel
-
-Follow **@AMUpdateChannel** on Twitter: https://twitter.com/AMUpdateChannel
+[App Manager Docs](https://muntashirakon.github.io/AppManager)
 
 ### Translations
 
-Translate **App Manager** at _Weblate_: https://hosted.weblate.org/engage/app-manager/
+Help translate [App Manager at Hosted Weblate](https://hosted.weblate.org/engage/app-manager/)
+The App Manager Docs are translated [at Crowdin](https://crwd.in/app-manager-docs) till https://github.com/MuntashirAkon/AppManager/issues/303 is closed.
```

Comment:
You don't have to explicitly mention about #303 here. I can post an announcement in the Crowdin project and the Telegram channel (like I did earlier for App Manager strings) before the migration.